### PR TITLE
Validation: Corrected date to fix yaml-header code

### PR DESCRIPTION
**Pull request by kturetz**
Corrected date in metadata in attempt to fix yaml-header-syntax error code.

### CODE REVIEW

**Comment by kturetz** [comment]
@yasong-Ashley 
@stevemunk 

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

**Comment by kturetz** [comment]
@yasong-Ashley
@stevemunk

I'm checking in on this PR to see if there are any issues that need to be resolved in order for it to be merged. Thanks!

File: BingMaps/rest-services/routes/optimized-itinerary.md [hunk 1]
```diff
@@ -1,17 +1,18 @@
 ---
-title: "Optimized Itinerary | Microsoft Docs"
-ms.custom: ""
-ms.date: "05/03/2019" Last updated "04/08/2020"
-ms.reviewer: ""
-ms.suite: ""
-ms.tgt_pltfrm: ""
-ms.topic: "article"
+title: Optimized Itinerary | Microsoft Docs
+description: The Bing Maps Multi-Itinerary Optimization API returns an itinerary schedule for one or more agents to travel between multiple itinerary items.
```
Comment (stevemunk) [inline BingMaps/rest-services/routes/optimized-itinerary.md:3]:
```suggestion
description: Learn how to use the Bing Maps Multi-Itinerary Optimization API that returns an itinerary schedule for one or more agents to travel between multiple itinerary items.
```

**Comment by kturetz** [comment]
#Sign-off

@stevemunk 

Please merge if ready! Thanks!

**Pull request merged**